src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/checkout_services/placeOrder.js

```diff
@@ -139,6 +142,26 @@ function placeOrder(req, res, next) {
 
     // Handles payment authorization
     var handlePaymentResult = adyenHelpers.handlePayments(order);
+
+    //Check if gift card was used
+    if(session.privacy.giftCardResponse) {
```

Comment:
Is this where you create the second payment instrument after order creation?

Reply:
yes